Makefile

```diff
@@ -162,11 +162,14 @@ shell-env: tmp/.shell-env
 
 # Note that we need Ekam to build node_modules before we can run Meteor, hence
 # the dependency on tmp/.ekam-run.
-tmp/.shell-env: tmp/.ekam-run $(IMAGES)
+tmp/.shell-env: tmp/.ekam-run $(IMAGES) CHANGELOG.md
 	@mkdir -p tmp
 	@touch tmp/.shell-env
 	@mkdir -p node_modules/capnp
 	@bash -O extglob -c 'cp src/capnp/!(*test*).capnp node_modules/capnp'
+	@echo '<template name="changelog">' > shell/client/changelog.html
```

Comment:
Can you write a separate make rule for shell/client/changelog.html and have tmp/.shell-env depend on it, as we do for images?